tests/main.test.ts

```diff
@@ -208,6 +208,25 @@ it(`should allow updating the pinned version using the "prepare" command`, async
   });
 });
 
+it(`should allow to call "prepare" with a tag`, async () => {
+  await xfs.mktempPromise(async cwd => {
+    await expect(runCli(cwd, [`prepare`, `--activate`, `npm@latest-7`])).resolves.toMatchObject({
+      exitCode: 0,
+      stderr: ``,
+    });
+
+    await xfs.writeJsonPromise(ppath.join(cwd, `package.json` as Filename), {
+      // empty package.json file
+    });
+
+    await expect(runCli(cwd, [`npm`, `--version`])).resolves.toMatchObject({
+      stdout: `7.24.2\n`,
```

Comment:
That's going to fail next time npm updates the `latest-7` tag, are we positive it's not going to change? If so, let's add a comment, ideally with a link to their announcement.

Reply:
Yes, `latest-7` will be updated.
However, I thought it was OK because the communication was recorded by nock. Is that not sufficient, and should I make sure it succeeds even if nock replay is disabled?

Reply:
On my own machine, I don't always put `NOCK_ENV=replay` in my env, so the test would start failing in that case. Hopefully we can come up with a more robust test. What do you think of

```js
    const npmVersion = await runCli(cwd, [`npm`, `--version`]);
    expect(npmVersion).toMatchObject({
      stderr: ``,
      exitCode: 0,
    });
    expect(npmVersion.stdout).toMatch(/^7\./);
```

Reply:
That's a reasonable solution!

Reply:
@merceyz Thank you! I'll fix it.

Reply:
https://github.com/nodejs/corepack/pull/176